### DISCUSSION TITLE
Nuxt 3 SSR application content in Multi Step form not rendering on Refresh?

### DISCUSSION THREAD

**Discussion by Kealtie** [formkit/formkit#714] [General]
Hi,

I've tried making a proof of concept of a multi-step form using FormKit in my Nuxt 3 SSR application. 

I soon came to the conclusion that when implementing the [Usage example](https://formkit.com/plugins/multi-step) the Form content wasn't showing. On the page source it was visible that the first step also had the 'hidden' property, I figured this might be a bug in FormKit?

**Comment by andrew-boyd** [comment]
Got a confirmed reproduction here — we'll look into what's causing it, certainly something SSR.

https://stackblitz.com/edit/nuxt-formkit-multistep?file=package.json,nuxt.config.ts,formkit.config.js,app.vue

It's not ideal, but for the time being you can bypass it by wrapping the MultiStep in `<ClientOnly>`.

**Comment by andrew-boyd** [comment]
This has been fixed in newer FormKit versions.
